deploy-as-code/helm/charts/municipal-services/echallan-services/values.yaml

```diff
@@ -98,6 +98,11 @@ env: |
       configMapKeyRef:
         name: egov-service-host
         key: collection-services
+  - name: BILLING.SERVICE.HOST
+    valueFrom:
+      configMapKeyRef:
+        name: egov-config
+        key: egov-services-fqdn-name
```

Comment:
Use [this](https://github.com/misdwss/iFix-DevOps/blob/mgramseva/deploy-as-code/helm/environments/mgramseva-qa.yaml#L130) config map reference.

Reply:
Done